lib/tests/parse.rs

```diff
@@ -1,26 +1,9 @@
 extern crate weldr;
 
-use weldr::{error::ResolveError, Command, FileRefResolver, SourceFileRef, SourceMap, SubFileRef};
+use weldr::{error::ResolveError, Command, FileRefResolver, SourceMap, SubFileRefCmd};
 
 use std::collections::HashMap;
 
-fn print_rec(source_map: &SourceMap, source_file_ref: SourceFileRef, indent: usize) {
```

Comment:
Can we leave this, if only for debugging? This is quite useful if I remember correctly.